### PR TITLE
Add post-hook script to tag created DynamoDB tables

### PR DESCRIPTION
**Pull request by jpopelka**
Related to https://github.com/openshiftio/openshift.io/issues/1081

### CODE REVIEW

**Comment by jpopelka** [comment]
Seems to be blocked by https://github.com/almighty/almighty-jobs/pull/450

**Comment by jpopelka** [comment]
[test]

File: openshift/template.yaml [hunk 1]
```diff
@@ -30,6 +30,14 @@ objects:
   spec:
     selector:
       service: bayesian-gremlin-${CHANNELIZER}
+    strategy:
+      type: Recreate
```
Comment (msrb) [inline openshift/template.yaml]: Do we really want this? What's wrong with RollingUpdate?

File: openshift/template.yaml [hunk 2]
```diff
@@ -30,6 +30,14 @@ objects:
   spec:
     selector:
       service: bayesian-gremlin-${CHANNELIZER}
+    strategy:
+      type: Rolling
+      recreateParams:
```
Comment (msrb) [inline openshift/template.yaml]: Looks like a leftover from `Recreate` strategy :)

**Comment by msrb** [comment]
Looks good, thanks. Quick question: have you tested this in the dev cluster or should I do that? :)

**Comment by jpopelka** [comment]
Just tested on dev cluster and seems to work OK.
Please merge https://github.com/fabric8-analytics/fabric8-analytics-common/pull/430 first.

**Review by msrb — APPROVED** [review]
LGTM, thanks 😉

**Pull request merged**